src/binding/class.rs

```diff
@@ -69,14 +73,15 @@ pub fn define_attribute(object: Value, name: &str, reader: bool, writer: bool) {
 pub fn respond_to(object: Value, method: &str) -> bool {
     let result = unsafe { class::rb_respond_to(object, symbol::internal_id(method)) };
 
-    util::c_int_to_bool(result)
+    c_int_to_bool(result)
 }
 
 pub fn define_method<I: Object, O: Object>(klass: Value, name: &str, callback: Callback<I, O>) {
     let name = util::str_to_cstring(name);
 
     unsafe {
-        class::rb_define_method(klass, name.as_ptr(), callback as CallbackPtr, -1);
+        let callback = callback as *const libc::c_void;
+        class::rb_define_method(klass, name.as_ptr(), callback, -1)
```

Comment:
@goyox86  I know this is a minor nitpick.  But is there a reason why the three instances of `callback as CallbackPtr` where changed here in this file and once in `src/binding/module.rs`?  If these don't need to be changed I think the should remain as they were.

Reply:
You are absolutely right! addressed in ae5896c078ec9d6abfc32a9882e3f14fa5b5a85d